### PR TITLE
Turn error from cqm-parsers into MeasureLoadingBadPackage error

### PR DESCRIPTION
**Pull request by lukoktonos**
This PR turns the error that cqm-parsers can raise into a more descriptive error message in bonnie in production mode.

Need to test in production mode with mailer disabled. Steps to test locally:

1. in `config/environments/production.rb` change the value of config.serve_static_files to true
2. `rm -rf tmp/cache/assets`
3. `rm -rf public/assets`
4. `bundle exec rake assets:precompile RAILS_ENV=production`
5. in `config/secrets.yml` copy the `development.secret_key_base`to the `production.secret_key_base`
6. in config/environments/production.rb, set `config.action_mailer.raise_delivery_errors = false` and comment out the other `config.action_mailer` lines.

Upload `CMS160_v5_2_Artifacts_with_json.zip` (from our JIRA test cycle)

Error message should match:

image

Pull requests into Bonnie require the following. Submitter and reviewer should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable ("N/A") and :white_check_mark:.

**Submitter:**
- [x] This pull request describes why these changes were made.
- [x] This PR is into the correct branch.
- [x] JIRA ticket for this PR: https://jira.mitre.org/browse/BONNIE-2107
- [x] JIRA ticket links to this PR
- [x] Code diff has been done and been reviewed (it **does not** contain: additional white space, not applicable code changes, debug statements, etc.)
- [x] If UI changes have been made, google WAVE plug-in has been executed to ensure no 508 issues were introduced.
- [x] Tests are included and test edge cases N/A it's in production mode and we don't have production mode tests
- [x] Tests have been run locally and pass (remember to update Gemfile when applicable)
- [x] Test fixtures updated and documented as necessary ( see [internal wiki](https://gitlab.mitre.org/bonnie/internal-documentation/wikis/testing#test-fixtures) )
- [x] Code coverage has not gone down and all code touched or added is covered. 
     * In rare situations, this may not be possible or applicable to a PR. In those situations:
         1. Note why this could not be done or is not applicable here: 
         2. Add TODOs in the code noting that it requires a test
         3. Add a JIRA task to add the test and link it here: 
- [x] Automated regression test(s) pass N/A

**Reviewer 1:**

Name: @zlister 
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

**Reviewer 2:**

Name: @hossenlopp 
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

### CODE REVIEW

**Review by zlister — APPROVED** [review]

File: app/helpers/measure_helper.rb [hunk 1]
```diff
@@ -217,6 +217,7 @@ def update_measure(uploaded_file:, target_id:, value_set_loader:, user:)
 
   def turn_exception_into_shared_error_if_needed(error)
     return error if error.is_a?(SharedError)
+    return MeasureLoadingBadPackage.new(error.inspect) if error.inspect.include? 'Verify the QDM version of the measure package is correct.'
```
Comment (hossenlopp) [inline app/helpers/measure_helper.rb:220]: This seems a bit of a rickety way to check the error type. Is the source error only a `StandardError`?

**Review by hossenlopp — APPROVED** [review]
Tested in production mode.
